src/app/workflow/dag/state/dag.service.ts

```diff
@@ -326,6 +326,9 @@ export class DagService {
     };
     const destroy = () => {
       try {
+        if (document.getElementsByClassName('dag-tooltip')[0]) {
```

Comment:
Are you always only expecting one element? You could do a loop to be safe, then you don't even need an if.